source/crud.txt

```diff
@@ -6,6 +6,13 @@ Perform CRUD Operations
 
 .. default-domain:: mongodb
 
+   .. facet::
+   :type: genre
+   :values: reference
+
+.. meta:: 
+   :keywords: server
```

Comment:
Thanks @sarahemlin, do we have a keyword for the Shell?

Reply:
The [Official Taxonomy](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy?src=contextnavpagetreemode) has shell as a programmingLanguage value.